client/lcd/certificates.go

```diff
@@ -43,7 +43,7 @@ func generateSelfSignedCert(host string) (certBytes []byte, priv *ecdsa.PrivateK
 		KeyUsage:              x509.KeyUsageKeyEncipherment | x509.KeyUsageDigitalSignature | x509.KeyUsageCertSign,
 		ExtKeyUsage:           []x509.ExtKeyUsage{x509.ExtKeyUsageServerAuth},
 		BasicConstraintsValid: true,
-		IsCA:                  true,
```

Comment:
Might want to revert this?

Reply:
It's from make format. I think it's fine to leave?

Reply:
strange...looks unformatted to me.

Reply:
huh your right, this does look unformatted. IDK the linter was throwing messages locally with how it used to be, and its no longer throwing messages.